code-challenges.js

```diff
@@ -12,6 +12,30 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+var num4 = "haha"
+// Expected output: "not a valid input."
+
+//created a function divBy3 that takes a number as an argument
+const divBy3 = (number) => {
+    //I am using an if else if chain to sort through conditions (i.e. if a number is evenly divisible by 3 or not)
+    //added the typeof condition to make sure only numbers go through the logic, the rest goes to the error catchall.
+    if (number % 3 === 0 && typeof number === "number"){
+        //return an interpolated string that says the number is evenly divisible
+        return `${number} is evenly divisible by three.`
+    } else if(number % 3 !== 0 && typeof number === "number"){
+        //return an interpolated string that says the number is not evenly divisible.
+        return `${number} is not evenly divisible by three.`
+    //catchall for errors
+    } else {
+        return `not a valid input.`
+    }
```

Comment:
♥️ LOVE this! Great use of typeof to really check if the number is actually a number for your error catching.